config/locales/en/summary.yml

```diff
@@ -350,3 +359,16 @@ en:
         question: Enter the holder number
         absence_answer: ''
       # END premium_bonds
+      #
+      # START investments
+      description: 
+        question: Describe the investment
+        absence_answer: *absence_none
+      value: 
+        question: What is the value of the investment?
+        absence_answer: *absence_none
+      ownership_type: 
+        question: Whose name is the investment in?
+        answers: *OWNERSHIP_TYPE 
```

Comment:
For the OWNERSHIP_TYPE options, the designs have it as `My client`

But also, I think they're also saying not to have the ownership row if partner ==.no

Reply:
Another good spot. Thank you. Will fix when extract for national savings.